helm/blueapi/templates/statefulset.yaml

```
@@ -52,6 +52,14 @@ spec:
         emptyDir:
           sizeLimit: 5Gi
       {{- end }}
+      {{- if .Values.debug.enabled }}
+      - name: home  # Required for vscode to install plugins
+        hostPath:
+          path: {{ .Values.debug.home }}
+      - name: nslcd  # Shared volume between main and sidecar container
+        emptyDir:
+          sizeLimit: 500Mi
```

```suggestion
          sizeLimit: 5Mi
```

nit: could be much smaller